src/ConstructiveSolidGeometry/Sampling.jl

```diff
@@ -0,0 +1,11 @@
+function sample_surface(c::AbstractVolumePrimitive{T}, sampling::Union{Real, NTuple{3,Int}}) where {T}
+    samples = CylindricalPoint{T}[]
+    for surf in get_decomposed_surfaces(c)
+        append!(samples, sample(surf, sampling))
+    end
+    samples
+end
+
+function sample_surface(c::AbstractSurfacePrimitive{T}, sampling::Union{Real, NTuple{3,Int}}) where {T}
+    sample(c, sampling)
+end
```

Comment:
just do 
```julia
sample_surface(c::AbstractSurfacePrimitive, sampling) = sample(c, sampling)
```
So you also parse other methods we might define later. E.g. if sampling is `1mm`.